options.go

```diff
@@ -81,6 +81,10 @@ type Options struct {
 
 	// When Stat is not nil, statistical logic needs to be executed
 	Stat *Statistics
+
+	// don't export branch node's key in snapshot, default to false.
+	// see: https://github.com/cosmos/iavl/issues/688
+	DisableExportBranchNodeKey bool
```

Comment:
is there any reason to expose this to an application

Reply:
I guess it's possible to control the migration process without doing a coordinated upgrade, for example release the binary first but disable the breaking behavior, after most of people updated binary, then enable this.